app/src/ui/check-runs/ci-check-run-list.tsx

```diff
@@ -63,7 +66,10 @@ export class CICheckRunList extends React.PureComponent<
   ): ICICheckRunListState {
     let checkRunExpanded = currentState?.checkRunExpanded ?? null
 
-    if (currentState === null || !currentState.hasUserToggledCheckRun) {
+    if (
+      (currentState === null || !currentState.hasUserToggledCheckRun) &&
+      !this.props.notExpandable
+    ) {
```

Comment:
Nit: Alternative? Is this better? 🤷  (might require linting)
```suggestion
    if (currentState?.hasUserToggledCheckRun !== true && !this.props.notExpandable) {
```